packages/cli/src/assert/assert.js

```diff
@@ -53,7 +57,7 @@ async function runCommand(options) {
   // If we have a budgets file, convert it to our assertions format.
   if (budgetsFile) options = await convertBudgetsToAssertions(readBudgets(budgetsFile));
 
-  const lhrs = loadSavedLHRs().map(json => JSON.parse(json));
+  const lhrs = loadSavedLHRs(options.lhr).map(json => JSON.parse(json));
```

Comment:
it looks like if we pass `options.lhr` it will be gone cause it was replaced by the `convertBudgetsToAssertions` returned value.

if my understanding is correct & this is a vallid issue, I would love to create PR to fix this

Reply:
Thanks! fixes coming right up

Reply:
The fixes are at #1062, @connorjclark.
Could you kindly review it at your convenience?